src/utils/editor.ts

```diff
@@ -222,7 +222,7 @@ export const extractMetadata = (body:string, thumbIndex?:number) => {
   const mUrls = extractUrls(body);
   const mUsers = body && body.match(userReg);
 
-  const matchedImages = extractImageUrls({urls:mUrls});
+  const matchedImages = extractImageUrls({urls:mUrls}).filter((element,index) => index < 10); //filter only first 10 images
```

Comment:
@aliseyalvi please test all use-cases of returned object `matchedImages` and test varying use cases of extraMetaData, I have a feeling it will be disturb some app functions

Reply:
@noumantahir apparently it is working well. Also tested thumb selection process and it is working fine as well.

Reply:
@aliseyalvi video reference. https://github.com/ecency/ecency-mobile/pull/2473#issuecomment-1261999538

other than this, why not use slice(), instead of using filter(), apparently slice method is less expensive compared to filter.